guide/14-deep-learning/how_pspnet_works_and_its_implementation_details_within_arcgis.ipynb

```diff
@@ -0,0 +1,287 @@
+{
```

Comment:
<blockquote>In the guide&nbsp;<a href="https://developers.arcgis.com/python/guide/how-unet-works/" target="_blank">How u-net works</a>, we have learned how to solve the problem of semantic segmentation using deep learning architecture called Unet using the ArcGIS Python API. There are many other semantic segmentation algorithms like PSPNet, Deeplab, etc. In this guide, we will mainly focus on Pyramid scene parsing network (PSPNet) [1] which is one of the most well-recognized image segmentation algorithms as it won ImageNet Scene Parsing Challenge 2016 and its paper has more than 1500 citations. PSPNet is another semantic segmentation model along with the Unet that has been implemented into the&nbsp;<code>arcgis.learn</code>&nbsp;module which can be trained to classify pixels in a raster.</blockquote><p> </p><p>In the guide&nbsp;<a href="https://developers.arcgis.com/python/guide/how-unet-works/" target="_blank">How u-net works</a>, we have learned in detail about semantic segmentation using U-net. There are many semantic segmentation algorithms such as U-net, PSPNet, Feature Pyramid Network (FPN), etc. In this guide,  we will focus on Pyramid scene parsing network (PSPNet) [1] which is one of the most well-recognized image segmentation algorithms as it won ImageNet Scene Parsing Challenge 2016 and its paper has more than 1500 citations. PSPNet is another semantic segmentation model along with the U-net that has been implemented into the&nbsp;<code><code>arcgis.learn</code></code>&nbsp;module which can be trained to classify pixels in a raster.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/547/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Done.</p>